src/accel/TimeOutput.hh

```diff
@@ -49,16 +47,16 @@ class TimerOutput final : public OutputInterface
     //!@}
 
     // Record the accumulated action times
-    void RecordActionTime(MapStrReal&& time);
+    void record_action_time(MapStrReal&& time);
```

Comment:
I guess this is more of a "power user" class now (anyone downstream who doesn't want to use the integration classes) but we need to differentiate more clearly what the line is between "G4-style" and "Celeritas style"

Reply:
Yeah I got 90% through and then wondered if I should have changed it... I'm happy to go either way.

Reply:
I'd say since it should be easy to revert it and it lowers the change count and we aren't sure... maybe do a quick reversion? Sorry for the extra work 🫤